db/schema.rb

```diff
@@ -17,77 +16,76 @@
   enable_extension "plpgsql"
 
   create_table "assignments", force: :cascade do |t|
-    t.string   "name",           null: false
-    t.jsonb    "metadata"
-    t.string   "workflow_id",    null: false
-    t.string   "subject_set_id", null: false
-    t.integer  "classroom_id",   null: false
-    t.datetime "created_at",     null: false
-    t.datetime "updated_at",     null: false
+    t.string "name", null: false
+    t.jsonb "metadata"
+    t.string "workflow_id", null: false
+    t.string "subject_set_id", null: false
+    t.bigint "classroom_id", null: false
```

Comment:
Strange. I get a completely difference schema.rb after doing this:

```
git checkout master
bin/rake db:reset
git checkout rails5-upgrade
bin/rake db:migrate
```

As of Rails 5, apparently it generates bigint primary keys. I'm wondering if you maybe started with an empty database when already on Rails 5, and then ran `db:migrate` rather than `db:setup` to initialize it?